### PR TITLE
Apply `wrapArguments` rule for `swiftformat`

### PR DESCRIPTION
**Pull request by cgrindel**

### CODE REVIEW

File: tools/generator/test/Fixtures.swift [hunk 1]
```diff
@@ -1543,8 +1543,7 @@ cp "${SCRIPT_INPUT_FILE_0}" "${SCRIPT_OUTPUT_FILE_0}"
             "E1": [
                 PBXSourcesBuildPhase(
                     files: buildFiles([PBXBuildFile(
-                        file: elements[.external("a_repo/a.swift")]!),
-                    ])
+                        file: elements[.external("a_repo/a.swift")]!)])
```
Comment (brentleyjones) [inline tools/generator/test/Fixtures.swift:1546]:
```suggestion
                        file: elements[.external("a_repo/a.swift")]!
                    )])
```

Comment (cgrindel) [reply tools/generator/test/Fixtures.swift]: Done

File: tools/generator/test/Fixtures.swift [hunk 2]
```diff
@@ -1543,8 +1543,9 @@ cp "${SCRIPT_INPUT_FILE_0}" "${SCRIPT_OUTPUT_FILE_0}"
             "E1": [
                 PBXSourcesBuildPhase(
                     files: buildFiles([PBXBuildFile(
-                        file: elements[.external("a_repo/a.swift")]!),
-                    ])
+                        file: elements[.external("a_repo/a.swift")]!
+                    )])
+                    // swiftlint:disable:previous trailing_comma
```
Comment (brentleyjones) [inline tools/generator/test/Fixtures.swift:1548]: We can have the trailing comma, I was mainly calling out the odd newline usage.

Comment (cgrindel) [reply tools/generator/test/Fixtures.swift]: I just applied a minimal change to add a comma. It looks a little funny, but avoids us having to break out the array literal into multiple lines. Let me know what you think.

Comment (brentleyjones) [reply tools/generator/test/Fixtures.swift]: I would prefer to have us break out the array versus this.

File: tools/generator/test/Fixtures.swift [hunk 3]
```diff
@@ -1562,8 +1562,8 @@ cp "${SCRIPT_INPUT_FILE_0}" "${SCRIPT_OUTPUT_FILE_0}"
             "E1": [
                 PBXSourcesBuildPhase(
                     files: buildFiles([PBXBuildFile(
-                        file: elements[.external("a_repo/a.swift")]!),
-                    ])
+                        file: elements[.external("a_repo/a.swift")]!
+                    )])
```
Comment (cgrindel) [inline tools/generator/test/Fixtures.swift:1566]: @brentleyjones I removed the weird comma for now. This passes `swiftformat`. We will deal with the `swiftlint` issues in subsequent PRs.

**Review by brentleyjones — APPROVED** [review]

**Pull request merged**